cfn/guardduty.template

```diff
@@ -285,7 +285,7 @@
                            "",
                            [
                                 "const AWS = require('aws-sdk');\n",
-                                "const response = require('cfn-response');\n",
+                                "const response = require('./cfn-response');\n",
```

Comment:
Not sure why that is necessary. cfn-response is in `node-modules` right?

I mean i'm sure youns did this for a reason. Its just seems like an odd requirement

Reply:
This is sort of a workaround unless AWS fixes it properly
https://github.com/aws/aws-sdk-js/issues/2955

Reply:
noted. thats a funky.